package.json

```diff
@@ -46,7 +45,7 @@
     "@playwright/test": "^1.45.1",
     "appium": "^2.4.1",
     "appium-uiautomator2-driver": "patch:appium-uiautomator2-driver@npm%3A3.7.3#~/patches/appium-uiautomator2-driver-npm-3.7.3-24e944f6c5.patch",
-    "appium-xcuitest-driver": "^7.26.0",
```

Comment:
not sure why a downgrade is needed?

Reply:
will revert this, forgot to do this after debugging the error earlier